### PR TITLE
Update weave-network-policy.md

### PR DESCRIPTION
**Pull request by dengyi1996**
add how to check if weave working correctly

> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.8 Features: set Milestone to `1.8` and Base Branch to `release-1.8`
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>
> NOTE: Please check the “Allow edits from maintainers” box (see image below) to
> [allow reviewers to fix problems](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) on your patch and speed up the review process.
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4812)

### CODE REVIEW

**Comment by zacharysarah** [comment]
@dengyi1996 :wave: Thanks for opening this PR. The technical content here is solid, but this PR requires significant revision before it can be merged. 

The language needs to follow Kubetnetes' [style guide best practices](https://kubernetes.io/docs/home/contribute/style-guide/), especially for clarity of language and accuracy of grammar. 

Let me know if you have any questions. When you're ready for another review, please mention me (`@zacharysarah`) in a comment. 🙇

**Comment by chenopis** [comment]
@zacharysarah You might want to give some specific examples of what should be revised.

**Comment by zacharysarah** [comment]
@dengyi1996 :wave: Apologies for the slow response. As a specific example for revision, consider lines [127-131](https://github.com/kubernetes/kubernetes.github.io/pull/4812/files#diff-b0dc5771cae9fbea1ca69fce28247d50R127):
```
you can refer to the previous text and create a Networkpolicy.And use
     ```shell
     kubectl logs -f  weave-net-pmw8w weave-npc  -n kube-system
     ```
get the logs.You will see that
```
To pass review, this section needs to look more like this:
```
After creating a NetworkPolicy, use kubectl to check the logs:
     ```shell
     kubectl logs -f  weave-net-pmw8w weave-npc  -n kube-system
     ```
Log output looks like this:
```
All other text in this PR needs similar improvement.

Does that help? If you have any questions, please feel free to reach out.

**On an related note:** The page needs some unrelated updates to make the example section display correctly on the site. (Currently, the example [doesn't display](https://kubernetes.io/docs/tasks/administer-cluster/weave-network-policy/).) It makes sense to add that change to this PR; I'm happy to make the updates, once we resolve the existing issues here.

**Comment by zacharysarah** [comment]
@dengyi1996 :wave: Any updates?

**Comment by dengyi1996** [comment]
＠zacharysarah 
thanks for your document and sugesstions,I will check it and modify soon.
By the way,shall I close this pr and open a new one?

**Comment by dengyi1996** [comment]
@zacharysarah I update this patch

File: docs/tasks/administer-cluster/weave-network-policy.md [hunk 1]
```diff
@@ -103,8 +99,80 @@ spec:
...
+weave-net-231d7                         2/2       Running   1          7d        10.2.0.17       worknodegpu
+weave-net-7nmwt                         2/2       Running   3          9d        192.168.2.131   masternode
+weave-net-pmw8w                         2/2       Running   0          9d        192.168.2.216   worknode2
+```
+
+Each Node has a weave Pod, and all Pods are `Running` and `2/2 READY`. (`2/2` means that each Pod has `weave` and `weave-npc`.)
+
+2. Create a Network Policy. 
+
+For more information, see "[this link]()".
```
Comment (zacharysarah) [inline docs/tasks/administer-cluster/weave-network-policy.md]: @dengyi1996 Please provide a link to the relevant documentation for creating a Network Policy.

File: docs/tasks/administer-cluster/weave-network-policy.md [hunk 2]
```diff
@@ -103,8 +99,80 @@ spec:
...
+INFO: 2017/08/14 02:22:32.511992 EVENT AddNetworkPolicy {"metadata":{"name":"aaa","namespace":"myns","selfLink":"/apis/extensions/v1beta1/namespaces/myns/networkpolicies/aaa","uid":"67b229fd-8097-11e7-92f3-005056a3bc75","resourceVersion":"1507955","generation":1,"creationTimestamp":"2017-08-14T02:22:22Z"},"spec":{"podSelector":{"matchExpressions":[{"key":"inns","operator":"In","values":["yes"]}]},"ingress":[{"from":[{"podSelector":{"matchExpressions":[{"key":"inns","operator":"In","values":["yes"]}]}}]}]}}
+INFO: 2017/08/14 02:22:32.512103 creating ipset: &npc.selectorSpec{key:"inns in (yes)", selector:labels.internalSelector{labels.Requirement{key:"inns", operator:"in", strValues:[]string{"yes"}}}, ipsetType:"hash:ip", ipsetName:"weave-[T]a=ETzaKA{o*muaFe:2IX(t"}
+INFO: 2017/08/14 02:22:32.538003 adding rule: [-m set --match-set weave-[T]a=ETzaKA{o*muaFe:2IX(t src -m set --match-set weave-[T]a=ETzaKA{o*muaFe:2IX(t dst -j ACCEPT]
+^[^C
+```
+
+4. Finally, check iptables rules,logs like that
+
+```shell
+command
```
Comment (zacharysarah) [inline docs/tasks/administer-cluster/weave-network-policy.md]: Provide a specific command to retrieve the given output.

**Comment by zacharysarah** [comment]
LGTM :shipit:

**Pull request merged**